tooling/minimize/tests/ub/tree_borrows/mutable_ref_child_write_frozen.rs

```diff
@@ -0,0 +1,15 @@
+//@ compile-flags: --minimize-tree-borrows
+
+// Check that a foreign read makes an active mutable reference frozen.
+// After that, writing to this mutable reference becomes invalid.
```

Comment:
```suggestion
// After that, writing to this mutable reference is UB.
```
same for all the other tests